src/universal/conf/app.conf

```diff
@@ -0,0 +1,23 @@
+# This is the base configuration file for the Mantis ETC client.
+
+# This where all the default settings are defined (this file is packaged within the executable).
+# It should always go at the top.
+include "application.conf"
+
+# Enable HTTPS
+mantis {
+  network {
+    rpc {
+      http {
+        mode = "https"
+
+        certificate-keystore-path = "conf/mantis.jks"
```

Comment:
Should HTTPS be pre-configured with these values or disabled by default so user can configure it on its own?

Reply:
Imo for now I'd just disable it and have the user configure it on its own, we can later add it for development's purposes or provide some guide on to how the user should configure it